src/Hidden/Hidden.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import t from 'prop-types'
+import cx from 'classnames'
+import * as types from '../types'
+import Box from '../Box/Box'
+import {useTheme} from '../Provider'
+
+const Hidden = ({showAt, inline = false, className, children, ...props}) => {
+  let displayPoints = ['none']
+  const theme = useTheme()
+  const breakpoints = theme.breakpoints
+  if (showAt.length > 0) {
```

Comment:
Give `showAt` a default value in the prop args as `undefined.length` throws an error. 

We could simply the logic too and get it down to one loop:

```
const {breakpoints} = useTheme()
let displayPoints = 'none'

if (showAt && showAt.length > 0) {
  displayPoints = Object.keys(breakpoints).map((key, idx) => 
    showAt.includes(idx) ? (inline ? 'inline':'block') : 'none'
  )
}
```

You may have already considered this but what do you think about also adding a `below` and `above` prop? It would simplify the common case where you want to hide at the smallest breakpoint but show at all other breakpoints above it...

So instead of...
```
<Hidden showAt={[1, 2, 3]} />
```
You could do...
```
<Hidden below={1} />
```
Or hide at the top two breakpoints only...
```
<Hidden above={2} />
```

We'd still keep the `showAt` prop for handling non-consecutive values like `[1, 3]`. Thoughts?